galley/pkg/server/components/processing_test.go

```diff
@@ -78,14 +78,11 @@ loop:
 			args.Insecure = false
 			args.AccessListFile = os.TempDir()
 		case 4:
-			args.Insecure = false
-			args.AccessListFile = "invalid file"
-		case 5:
 			args.SinkAddress = "localhost:8080"
 			args.SinkAuthMode = "foo"
-		case 6:
+		case 5:
```

Comment:
I'm not sure how we weren't failing the original case 4 before. I'm getting the following error on master.  The accesslist file loading code will return a default blacklist accesslist if the file is invalid. It's technically not an error.

```bash
$ go test ./galley/pkg/server/components/... -run TestProcessing_StartErrors
2020-01-28T06:50:10.036558Z     info    processing      Reloaded mesh config (path: /tmp/TestProcessing_StartErrors723098988/meshcfg.yaml): 
2020-01-28T06:50:10.045886Z     info    processing      Reloaded mesh config (path: /tmp/TestProcessing_StartErrors325838206/meshcfg.yaml): 
2020-01-28T06:50:10.053212Z     info    processing      Reloaded mesh config (path: /tmp/TestProcessing_StartErrors678574789/meshcfg.yaml): 
2020-01-28T06:50:10.060456Z     info    processing      Reloaded mesh config (path: /tmp/TestProcessing_StartErrors704194400/meshcfg.yaml): 
2020-01-28T06:50:10.063221Z     info    processing      Runtime.run: Starting new session id:1
2020-01-28T06:50:10.063235Z     info    processing      session[1] "inactive" => "starting"
--- FAIL: TestProcessing_StartErrors (0.03s)
    processing_test.go:99: 0) err: err0
    processing_test.go:99: 1) err: err1
    processing_test.go:99: 2) err: err2
    processing_test.go:99: 3) err: unable to read access list file "/tmp": read /tmp: is a directory
    processing_test.go:98: 
        Expected
            <nil>: nil
        not to be nil
FAIL
FAIL    istio.io/istio/galley/pkg/server/components     0.355s
FAIL
```